.github/workflows/bazel_lint.yml

```diff
@@ -0,0 +1,36 @@
+#
+#  Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name: bazel_lint
+on: 
+  push:
+    paths:
+      - '**BUILD' 
```

Comment:
Would '/XXBUILD' match here too? I'd assume so based on https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#patterns-to-match-file-paths

Perhaps use something like: `WORKSPACE`, `**/BUILD`, `**/BUILD.bazel`, `**.bzl`.